plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -458,6 +458,19 @@ func ClusterRoles() []rbac.ClusterRole {
 				eventsRule(),
 			},
 		},
+		{
+			// a role for csi external components {csi-provisioner,csi-attacher,driver-registrar}
+			ObjectMeta: metav1.ObjectMeta{Name: "system:csi-external-components"},
```

Comment:
Roles are typically singular

Reply:
Where does the driver-registrar run?

If the provisioner, attacher, and driver are run in separate locations with different levels of access to the API, three roles might make more sense. I wouldn't want every kubelet to have the ability to delete any PV object

Reply:
In the test we are attempting to fix they are all run in the same location, however we wish to leave the flexibility of CSI Driver implementers to split them up if they wish. I will create 3 roles with minimal permissions required for each component